lib/vega.component.js

```diff
@@ -52,11 +65,53 @@
           d3.event.preventDefault();
         })
         .text('Open in Vega Editor');
+
+      if (params.length) {
+        ctrl = d3.select(id).append('div')
+          .attr('class', 'aside')
+          .append('p');
+
+        for (var i = 0, len=params.length, p, t; i<len; ++i) {
+          p = params[i];
+          t = p.type;
+
+          if (t.match('num')) {
+            var range = t.replace('num', '')
+              .replace(/[\[\]]/g, '')
+              .trim()
+              .split('-');
+
+            ctrl.append('label')
+              .text(p.name)
+              .append('input')
+                .attr('id', id.replace('#', '')+'_'+p.name)
+                .attr('type', 'range')
+                .attr('min', range[0])
+                .attr('max', range[1])
+                .attr('value', p.value);
+
+            ctrl.append('br');
+          }
+        }
+      }
+
+      view = chart({el: id}).update();
```

Comment:
Should be ok, `view` is declared [within the closure scope](https://github.com/vega/vega-tutorials/pull/1/files#diff-3e6ff52a3cf1460779d83eaf30113948R17).